test/test_site/index.md

```diff
@@ -95,6 +95,24 @@ head: myCustomHead.md, myCustomHead2.md
 
 ## <include src="testTrimInclude.md" trim inline />
 
+# Include with custom variables
+
+<include src="testIncludeVariables.md">
+  <span id="includedVariable">Included variable</span>
+  <span id="includedVariableWithMarkdown">__**Included variable with markdown**__</span>
+  <span id="includedVariableAsAttribute">color: blue</span>
+  <span id="includedVariableAsHtmlElement"><span>Included variable within html element</span></span>
+  <span id="includedVariableGlobalOverridden">Included variable overridden by global variable</span>
+  <span id="includedVariableInnerOverridden">Included variable overriding inner variable</span>
+  <span id="includedVariableInIncludedFile">Included variable in included file</span>
```

Comment:
`includedVariableInIncludedFile` -> `includedVariableInOuterIncludedFile` (which include is the value coming from is not very obvious with the names).